index.html

```diff
@@ -169,7 +169,7 @@ <h3 data-i18n="dgc-fun-currency" class="text-center text-uppercase text-secondar
                         <br>
                         <a class="btn btn-secondary mb-2" target="_blank" href="https://discord.gg/dogecoin">
                             <i class="fab fa-discord mr-2"></i>
-                            <span>Discord Community</span>
+                            <span data-i18n="dgc-discord-community">Discord Community</span>
```

Comment:
This needs to be changed again in a rebase. I think it's just dgc-discord now.

Reply:
So that means translation at https://github.com/dogecoin/dogecoin.com/pull/184/files#diff-ae095573d46324cb0b513edc704bdd1178a2d5b44aafd339e706399a1b306e72R260 should be removed right?

Reply:
Can just be renamed

Reply:
Okay :+1: